src/libraries/System.Text.RegularExpressions/src/System/Text/RegularExpressions/srm/matcher/SymbolicRegexMatcher.cs

```diff
@@ -493,6 +512,13 @@ private State<S> CreateNewTransition(State<S> q, S atom, int offset)
         /// </summary>
         public Match FindMatch(bool isMatch, string input, int startat = 0, int endat = -1)
         {
+            _checkTimeout = (System.Text.RegularExpressions.Regex.InfiniteMatchTimeout != _matchTimeout);
```

Comment:
We only use underscores for instance fields (https://github.com/dotnet/runtime/blob/main/docs/coding-guidelines/coding-style.md)
But, if this is the normal style in the DFA code already, there's no need to change it all today, as long as it' consistent. It's not important at this point.

Reply:
In this case, _checkTimeout is in fact an instance field. I'll move its initialization into the constructor.
We will soon do a cleanup phase, going over the codebase and fixing the coding style consistently. 
Right now it is not consistent (but mixed ... as a leftover of various styles).